To: vim_dev@googlegroups.com
Subject: Patch 7.3.384
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.384
Problem:    Mapping CTRL-K in Insert mode breaks CTRL-X CTRL-K for dictionary
	    completion.
Solution:   Add CTRL-K to the list of recognized keys. (James McCoy)
Files:	    src/edit.c


*** ../vim-7.3.383/src/edit.c	2011-10-26 17:04:23.000000000 +0200
--- src/edit.c	2011-12-23 12:36:22.000000000 +0100
***************
*** 2183,2189 ****
  		    || c == Ctrl_I || c == Ctrl_D || c == Ctrl_P
  		    || c == Ctrl_N || c == Ctrl_T || c == Ctrl_V
  		    || c == Ctrl_Q || c == Ctrl_U || c == Ctrl_O
! 		    || c == Ctrl_S || c == 's');
  	case CTRL_X_SCROLL:
  	    return (c == Ctrl_Y || c == Ctrl_E);
  	case CTRL_X_WHOLE_LINE:
--- 2183,2189 ----
  		    || c == Ctrl_I || c == Ctrl_D || c == Ctrl_P
  		    || c == Ctrl_N || c == Ctrl_T || c == Ctrl_V
  		    || c == Ctrl_Q || c == Ctrl_U || c == Ctrl_O
! 		    || c == Ctrl_S || c == Ctrl_K || c == 's');
  	case CTRL_X_SCROLL:
  	    return (c == Ctrl_Y || c == Ctrl_E);
  	case CTRL_X_WHOLE_LINE:
*** ../vim-7.3.383/src/version.c	2011-12-23 12:46:56.000000000 +0100
--- src/version.c	2011-12-23 12:53:38.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     384,
  /**/

-- 
Q: Is selling software the same as selling hardware?
A: No, good hardware is sold new, good software has already been used by many.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
